airbyte-integrations/connectors/source-square/source_square/source.py

```diff
@@ -329,21 +390,51 @@ class Orders(SquareStreamPageJson):
     data_field = "orders"
     http_method = "POST"
     items_per_page_limit = 500
+    cursor_field = "updated_at"
+    filterbody = """{{
+                        "filter": 
+                        {{
+                            "date_time_filter": 
+                            {{   
+                                "updated_at": 
```

Comment:
You are using `updated_at` here and `created_at` above, can we use the same cursor logic in both streams?

Reply:
Ideally we should use updated at. Order has sort field as updated at. Customer don't have sort by updated at. Hence using the created_at field.